.travis.yml

```diff
@@ -16,13 +19,17 @@ install:
   - git clone https://github.com/nf-core/tools.git /tmp/nf-core-tools
   - cd /tmp/nf-core-tools
   - pip install --user -e .
-  # Reset
+  # Make test directories
+  - mkdir ${TRAVIS_BUILD_DIR}/tests
   - cd ${TRAVIS_BUILD_DIR}/tests
 
 env:
-  - NXF_VER=0.27.6
-  - ''
```

Comment:
Need the empty string to try testing with the latest version of Nextflow, in addition to the minimum required.